Airwallex/Core/Sources/AWXUtils.m

```diff
@@ -76,6 +76,21 @@ - (NSString *)stringByRemovingIllegalCharacters {
     return [components componentsJoinedByString:@""];
 }
 
+- (NSString *)stringByInsertingBetweenWordsWithString:(NSString *)separator {
```

Comment:
Can we have some individual tests for this util?

Reply:
Yep, tests added